### PR TITLE
Removed vars

### PR DESCRIPTION
**Pull request by Michal78900**
Using vars in a language like C# is a bad practice and may leed to confusion even if property is named propellery. The easiest way to not have problems later is always specifing the type of a variable explicitily.

### CODE REVIEW

File: NwPluginAPI/Commands/ListCommand.cs [hunk 1]
```diff
@@ -1,3 +1,5 @@
+using PluginAPI.Core;
```
Comment (moddedmcplayer) [inline NwPluginAPI/Commands/ListCommand.cs:1]: Inside namespace

File: NwPluginAPI/Commands/ReloadCommand.cs [hunk 2]
```diff
@@ -1,3 +1,5 @@
+using PluginAPI.Core;
```
Comment (moddedmcplayer) [inline NwPluginAPI/Commands/ReloadCommand.cs:1]: Same here

**Review by moddedmcplayer — COMMENTED** [review]
Move usings inside namespaces (pro tip: `File | Settings | Editor | Code Style | C# | Code Style | Reference Qualification | Add 'using' directive to deepest scope`)

**Comment by sirmeepington** [comment]
Using `var` is correct if the type of the right hand assignment is self explanatory. This is unnecessary.

**Comment by Michal78900** [comment]
> Using `var` is correct if the type of the right hand assignment is self explanatory. This is unnecessary.

I agree, however not everywhere it's self explainatory
obraz
For example here you don't know what type of this variable it is, unless you worked with it before.

**Pull request closed without merging**